common/tests/conftest.py

```diff
@@ -232,6 +232,18 @@ def notification_template_enrolment_approved_en():
         Custom message: {{ custom_message }}
         {% endif %}
 """,
+        body_html="""
```

Comment:
One possibility would be to have a separate file for template. It might be clearer and easier to maintain.